android/automation.md

```diff
@@ -0,0 +1,175 @@
+
+### ✅ Active: Release builds
+
+Whenever a release is created on GitHub we build releases versions of Focus and Klar on taskcluster.
+
+* [Configuration](https://github.com/mozilla-mobile/focus-android/blob/master/.taskcluster.yml#L91)
+
+### 📓 Planned: Signing
+
+Signing requires access to Mozilla's signing certificates. In addition to that a signing task needs to be able to validate the origin of the APK ("Can I trust the task that built the APK?") to avoid signing random tasks. To get this task up and running we need help from the release engineering team.
+
+Currently this is done manually by the release engineering team after we open a signing request bug on Bugzilla.
+
+* [Chain of Trust](http://scriptworker.readthedocs.io/en/latest/chain_of_trust.html)
+* [pushapkscript](https://github.com/mozilla-releng/pushapkscript)
+
+### 📓 Planned: Upload to archive.mozilla.org
+
+We want to publish every release on archive.mozilla.org. Currently we upload those builds manually. In automation this will be done by  the beetmover_scriptworker.
+
+Before we can automate this task we need to automate signing builds and have a chain of trust.
+
+* [beetmover script](https://github.com/mozilla-releng/beetmoverscript/)
+* [beetmover configuration for Fennec](https://github.com/mozilla-releng/beetmoverscript/blob/master/beetmoverscript/templates/fennec_nightly.yml)
+
+### 📓 Planned: Release signoff
+
+This is a dummy task that does nothing except waiting on somebody to complete it. After the build is tested and the release stakeholders agree to ship, someone from the Focus team will use the taskcluster-cli tool to manually resolve this task.
+
+* [taskcluster ci](https://github.com/taskcluster/taskcluster-cli)
+
+### 📓 Planned: Publish
+
+Finally we want to publish a release build on Google Play
+
+Like other scriptworker instances, we can use the existing instance. The current implementation is fennec-specific. We have to add support there too. pushapk_scriptworker mainly delegates checks and uploads to mozapkpublisher. That tool needs Focus support. Working on mozapkpublisher doesn't require any Taskcluster knowledge.
+
+* [pushapkscript](https://github.com/mozilla-releng/pushapkscript)
+* [mozapkpublisher](https://github.com/mozilla-releng/mozapkpublisher)
+
+## Nightly (Alpha) pipeline
+
+We want to build Nightly versions from master. Those Nightly builds should be distributed via the Google Play Alpha channel and replace distributing via buddybuild. Updates via buddybuild require manual download and installation of updates. With Google Play builds can update automatically in the background.
+
+The Alpha channel is not intended to be public like the Beta channel. However it should be open to the L10N teams and interested contributors.
+
+The pipeline for Nightly builds looks like the release sign-off pipeline without the upload and sign-off tasks.
```

Comment:
This section is heavily outdated and wrong now. Nowadays Google Play supports custom tracks and that's what we use for Nightly. We created a Nightly track.

Reply:
I replaced this section with a todo.